src/test/java/org/javarosa/core/util/test/DataUtilTest.java

```diff
@@ -41,4 +43,23 @@ public void intersectionTest() {
         assertFalse(intersectSet.contains("two"));
         assertFalse(intersectSet.contains("three"));
     }
+
+    @Test
+    public void splitOnSpacesTest() {
```

Comment:
nit: would be good to add a test case demonstrating what happens when multiple spaces are added consecutively in between words.